modules/identities/internal/mappable/identity.go

```diff
@@ -59,3 +61,55 @@ func NewIdentity(id types.ID, immutableProperties types.Properties, mutablePrope
 		HasMutables:   baseTraits.HasMutables{Properties: mutableProperties},
 	}
 }
+
+func (identity identity) IsProvisioned(address sdkTypes.AccAddress) bool {
+	flag := false
+	accAddressListData, ok := identity.GetAuthentication().GetFact().(types.ListData)
+
+	if !ok {
+		panic(errors.IncorrectFormat)
+	}
+
+	if address.Empty() && !accAddressListData.IsPresent(base.NewAccAddressData(address)) {
+		flag = true
+	}
+
+	return flag
+}
```

Comment:
apply comments in this method every other method in this file

Reply:
Ok